xonsh/aliases.py

```diff
@@ -73,6 +73,19 @@ def source_bash(args, stdin=None):
     return
 
 
+def exec(args, stdin=None):
```

Comment:
Yes it would.  It would work the same way that the `source-bash` alias works now.  Officially we support a generic name that doesn't collide with anything.  However, people can override other names (if they want to) in their xonshrc by renaming their aliases.